CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Okta Node SDK Changelog
 
+## 1.2.0
+
+- Adds a default request executor that will retry requests when rate limit exceptions have occurred.  Please see the [README](README.md) for full documentation.  This executor is opt-in for now, but will be used by default in the 2.0 release.
```

Comment:
Would it make more sense to link directly to the request executer documentation?

Reply:
Just avoiding broken links when the TOC changes (as it always does)